src/lib/Microsoft.Health.Logger/Telemetry/TelemetryExtensions.cs

```diff
@@ -6,18 +6,32 @@
 using System;
 using EnsureThat;
 using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
 using Microsoft.ApplicationInsights.Metrics;
 
-namespace Microsoft.Health.Fhir.Ingest.Telemetry.Metrics
+namespace Microsoft.Health.Logging.Telemetry
```

Comment:
Note: It's kind of weird how the collective diff is shown here. Here's a more accurate description of the changes:
- `src/lib/Microsoft.Health.Logger/Telemetry/Metrics/MetricExtensionMethods.cs` was renamed (i.e. merged into `TelemetryExtensions.cs`) in commit 60ea06e
- `src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/MetricExtensionMethods.cs` was deleted in commit aa3a937